app/lib/actions/pulp/abstract_async_task_group.rb

```diff
@@ -31,6 +32,23 @@ def run(event = nil)
         end
       end
 
+      def cancel!
+        cancel
+        self.external_task = poll_external_task
+        # We suspend the action and the polling will take care of finding
+        # out if the cancelling was successful
+        suspend unless done?
+      end
+
+      def cancel
+        group = output['pulp_task_group']
+        if group
+          task_resource.cancel(group['id'])
+        end
```

Comment:
We don't for normal tasks:  https://github.com/Katello/katello/blob/master/app/lib/actions/pulp/abstract_async_task.rb#L87

I think its a valid question in both cases, but I don't know that it helps?  if it wasn't defined for some reason and you clicked cancel, nothing would happen the task would keep going (assuming it was).  I believe currently with dynflow if you threw an exception during cancel it would treat the task as failed and may pause it, which i don't think is desirable?